runtime/src/weights/pallet_teeracle.rs

```diff
@@ -18,35 +18,35 @@
 // --heap-pages=4096
 // --output=runtime/src/weights/pallet_teeracle.rs
 
-
 #![cfg_attr(rustfmt, rustfmt_skip)]
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 
 use frame_support::{traits::Get, weights::Weight};
 use sp_std::marker::PhantomData;
 
-/// Weight functions for pallet_teeracle.
+/// Weight functions for `pallet_teeracle`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_teeracle::WeightInfo for WeightInfo<T> {
-	// Storage: Teerex Whitelist (r:1 w:1)
+	// Storage: Teerex EnclaveIndex (r:1 w:0)
+	// Storage: Teerex EnclaveRegistry (r:1 w:0)
```

Comment:
Update exchange rate doesn't read the Teerex EnclaveRegistry. It only check if the enclave is in the EnclaveIndex

Reply:
we have a somewhat reasonable value now. will accept it as is